app/styles/ui/_diff.scss

```
@@ -855,6 +855,16 @@
       }
     }
   }
+
+  // When focus moves to the virtualized grid, we want to show a focus ring.
+  &:has(.ReactVirtualized__Grid:focus) {
```

```suggestion
  &:has(.ReactVirtualized__Grid:focus-visible) {
```
The above will make it so that focus ring only appears when you tab and focus it, and not when you click in the grid. That way we don't have the focus ring there when it doesn't need to be there.